api/v1alpha1/nutanix_node_types.go

```diff
@@ -101,6 +105,9 @@ func (NutanixMachineDetails) VariableSchema() clusterv1.VariableSchema {
 					Description: "systemDiskSize is size (in Quantity format) of the system disk of the VM eg. 20Gi",
 					Type:        "string",
 				},
+				"project": NutanixResourceIdentifier{}.VariableSchemaFromDescription(
```

Comment:
some suggestions from docs on text:
https://opendocs.nutanix.com/capx/v1.3.x/types/nutanix_machine_template/

      # Adds the cluster virtual machines to a project defined in Prism Central.
      # Replace NUTANIX_PROJECT_NAME with the correct project defined in Prism Central
      # Note: Project must already be present in Prism Central. <=== we should add "existing" in the description as well